projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js

```diff
@@ -72,8 +72,13 @@ export default function ProductManagementInspectorControl() {
 			success => {
 				setApiState( API_STATE_NOT_REQUESTING );
 				if ( success ) {
+					const defaultTitle = getMessageByProductType( 'default new product title', productType );
 					setPrice( 5 );
-					setTitle( '' );
+					setTitle( defaultTitle );
+					setInterval( '1 month' );
+					setIsMarkedAsDonation( false );
+					setIsCustomAmount( false );
+					setCurrency( 'USD' );
```

Comment:
Since these values are the same as the default ones used to set the initial state of the component maybe we could refactor these values to constants so that we can guarantee that the initial state and default state are always equal.

  Maybe something like this:
````

diff --git a/projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js b/projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js
index 25acc2b634..8c26e62d25 100644
--- a/projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js
+++ b/projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js
@@ -27,8 +27,15 @@ import { getMessageByProductType } from './utils';
 import { CURRENCY_OPTIONS } from '../../currencies';
 import { store as membershipProductsStore } from '../../../store/membership-products';
 
+const DEFAULT_CURRENCY = 'USD';
+const DEFAULT_PRICE = 5;
+const DEFAULT_INTERVAL = '1 month';
+const DEFAULT_IS_MARKED_AS_DONATION = false;
+const DEFAULT_IS_CUSTOM_AMOUNT = false;
+
 export default function ProductManagementInspectorControl() {
 	const { productType, setSelectedProductId } = useProductManagementContext();
+	const DEFAULT_TITLE = getMessageByProductType( 'default new product title', productType );
 	const { shouldUpgrade, siteSlug } = useSelect( select => {
 		const { getShouldUpgrade, getSiteSlug } = select( membershipProductsStore );
 		return {
@@ -39,14 +46,12 @@ export default function ProductManagementInspectorControl() {
 	const { saveProduct } = useDispatch( membershipProductsStore );
 
 	const [ apiState, setApiState ] = useState( API_STATE_NOT_REQUESTING );
-	const [ title, setTitle ] = useState(
-		getMessageByProductType( 'default new product title', productType )
-	);
-	const [ currency, setCurrency ] = useState( 'USD' );
-	const [ price, setPrice ] = useState( 5 );
-	const [ interval, setInterval ] = useState( '1 month' );
-	const [ isMarkedAsDonation, setIsMarkedAsDonation ] = useState( false );
-	const [ isCustomAmount, setIsCustomAmount ] = useState( false );
+	const [ title, setTitle ] = useState( DEFAULT_TITLE );
+	const [ currency, setCurrency ] = useState( DEFAULT_CURRENCY );
+	const [ price, setPrice ] = useState( DEFAULT_PRICE );
+	const [ interval, setInterval ] = useState( DEFAULT_INTERVAL );
+	const [ isMarkedAsDonation, setIsMarkedAsDonation ] = useState( DEFAULT_IS_MARKED_AS_DONATION );
+	const [ isCustomAmount, setIsCustomAmount ] = useState( DEFAULT_IS_CUSTOM_AMOUNT );
 
 	const intervalOptions = [
 		{ label: __( 'Month', 'jetpack' ), value: '1 month' },
@@ -72,13 +77,12 @@ export default function ProductManagementInspectorControl() {
 			success => {
 				setApiState( API_STATE_NOT_REQUESTING );
 				if ( success ) {
-					const defaultTitle = getMessageByProductType( 'default new product title', productType );
-					setPrice( 5 );
-					setTitle( defaultTitle );
-					setInterval( '1 month' );
-					setIsMarkedAsDonation( false );
-					setIsCustomAmount( false );
-					setCurrency( 'USD' );
+					setPrice( DEFAULT_PRICE );
+					setTitle( DEFAULT_TITLE );
+					setInterval( DEFAULT_INTERVAL );
+					setIsMarkedAsDonation( DEFAULT_IS_MARKED_AS_DONATION );
+					setIsCustomAmount( DEFAULT_IS_CUSTOM_AMOUNT );
+					setCurrency( DEFAULT_CURRENCY );
 				}
 			}
 		);

````